RELEASING.md

```diff
@@ -0,0 +1,244 @@
+      If you find any of these, you might want to correct the issues and
+      generate the files again. Once you're done, commit the changes.
+
+   3. Push the `.pot` files generated in step 2 to Transifex. To push only the
+      ones that changed, give a comma-separated list:
+
+      ```sh
+      cd po/
+      tx push -s -r musicbrainz.server,musicbrainz.instruments
+      ```
+
+      If you want to push languages, keep in mind you need to push
+      `musicbrainz.languages-9` (not `musicbrainz.languages`).
+
+   4. Push the commits from steps 1 & 2 to `master`.
+
+2. On the git branches merge step, to update the `beta` branch with the changes from the `master` branch,
+   merge `master` into `beta` (with `git merge --log=876423 --no-ff master`) and push to `beta`.
+   (Skip this, of course, if you're just deploying changes pushed directly to the `bet`a branch.)
+
+3. On the [build Docker images step](#build-docker-images),
+   enter `beta` for _IMAGE_BRANCH_ when doing “Build with Parameters”.
+   Wait until the build has completed.
+
+4. On the [deployment’s announcement step](#announce-the-deployment),
+   [set the banner message on beta](https://beta.musicbrainz.org/admin/banner/edit) to
+
+   ```html
+   Beta website is being updated, slowdowns may occur for a few minutes, thanks for your patience.
+   ```
+
+5. On the [deployment step](#deploy-to-production) itself,
+   run `./script/update_containers.sh beta`. Wait until the deployment has completed.
+
+   Notes:
+   - Background task runners follow the `production` branch only,
+     which means that new reports are not available on the beta website for now.
+   - The banner’s website is updated only after updating tickets; see below.
+
+6. On the [Jira step](#update-jira), set all the tickets
+   [`status = "In Development Branch" and project = MBS`](http://tickets.musicbrainz.org/secure/IssueNavigator.jspa?reset=true&jqlQuery=status+%3D+%22In+Development+Branch%22+and+project+%3D+mbs)
+   as status "In Beta Testing." For tickets which fixed a beta-only issue not
+   present in production, close the ticket as fixed and set the fix version
+   to "Beta." Make sure all the tickets have a type that makes sense, enough information
+   to be understandable, and the appropriate components set.
+
+   Notes: None of the following steps are involved in releasing beta:
+   - Blog
+   - Add git tag
+   - Release `musicbrainz-docker`
+
+7. Again, [set the banner message on beta](https://beta.musicbrainz.org/admin/banner/edit) to
+
+   ```html
+   Beta MusicBrainz Server has been updated on Month DD, see the list of <a href="https://tickets.metabrainz.org/issues/?filter=10715">tickets available for beta testing</a>.
+   ```
+## Update test
+
+Unlike production and beta, test is more flexible:
+- The `test` branch is not protected and can be force-pushed,
+  even though you have to check in with other developers that may be using it.
```

Comment:
```suggestion
  but please make sure no other developer is using it and ask for permission if they are.
```

Reply:
> Maybe something like this?

Genuine question: Is the original sentence not proper English?

> If you can figure out that the code being tested is already in production, for example, there's no need to ask

It is hard to figure out because the test container can be manually patched too, so it is not necessarily running the exact same code as in the `test` branch. So better ask in any case.

Reply:
Well, it's not incorrect English, but I think it's a lot more clear to be more explicit, especially if you think asking should be a rule :) Maybe:

```suggestion
  but you should always check with the other developers to make sure they are not currently using it.
```

Reply:
Thank you! Committed as https://github.com/metabrainz/musicbrainz-server/pull/2340/commits/a42d7f05bbd17fc0f28146100402c7642455e326